Tests/StreamChatTests/Repositories/AuthenticationRepository_Tests.swift

```diff
@@ -407,7 +410,7 @@ final class AuthenticationRepository_Tests: XCTestCase {
         XCTAssertEqual(refreshTokenCompletionCalls, 1)
         XCTAssertEqual(delegate.newStateCalls, 3)
         XCTAssertEqual(delegate.newState, .newToken)
-        XCTAssertEqual(delegate.clearDataCalls, 1)
+        XCTAssertEqual(delegate.logoutCallCount, 2)
```

Comment:
Is it expected to call logout twice?

Reply:
Good catch. We should only be logging out once, when the user changes. It is a test error, though